api/models.py

```diff
@@ -1836,6 +1836,7 @@ class OrgTypes(models.TextChoices):
     phone_number = models.CharField(verbose_name=_("phone number"), blank=True, null=True, max_length=100)
     last_frontend_login = models.DateTimeField(verbose_name=_("last frontend login"), null=True, blank=True)
     accepted_montandon_license_terms = models.BooleanField(verbose_name=_("has accepted montandon license terms?"), default=False)
+    limit_access_to_guest = models.BooleanField(verbose_name=_("is not guest user"), default=True)
```

Comment:
Let's also add a help_text providing information about how it is explicitly denies user most of the platform permissions.